pkg/cli/aws/aws.go

```diff
@@ -40,14 +40,22 @@ type CredentialOptions struct {
 // DefaultCredentialOptions uses the default role and profile
 // for accessing AWS.
 func DefaultCredentialOptions() *CredentialOptions {
+	return DefaultCredentialOptionsWithLog(logrus.New())
+}
+
+// DefaultCredentialOptionsWithLog uses the default role and profile
+// for accessing AWS.
+func DefaultCredentialOptionsWithLog(log logrus.FieldLogger) *CredentialOptions {
 	b, err := box.LoadBox()
 	if err != nil {
+		log.Errorf("Could not load box config: %w", err)
```

Comment:
suggestion: I forgot that this was logrus and not the gobox log module.
```suggestion
		log.WithError(err).Error("Could not load box config")
```